coa-helm/templates/cbas-api.yaml

```diff
@@ -34,7 +34,7 @@ data:
   {{ .Values.cbas.conf_file }}: |-
     spring:
       datasource:
-        {{- if .Values.postgres.podLocalDatabaseEnabled }}
+        {{- if eq ("true") (include "podLocalDatabaseEnabledForCromwellCBAS" .) }}
```

Comment:
Do we need an `if eq (true) (enabled)` or can we just do `if (enabled)`?

Reply:
You would think so! It seems that Helm helper functions like these cannot directly output booleans, very annoying. 

https://github.com/helm/helm/issues/10382

Reply:
Source: https://github.com/hashicorp/consul-template/blob/main/docs/templating-language.md